frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -101,17 +103,68 @@ export function ResultsQuery({
+    const { experimentResultsLoading, experimentResultCalculationError, experimentInsightType } =
+        useValues(experimentLogic)
+
+    function ChecklistItem({ failureReason, checked }: { failureReason: string; checked: boolean }): JSX.Element {
+        const failureReasonToText = {
+            'no-events': 'Events have been received',
+            'no-flag-info': 'Feature flag information is present on the events',
+            'no-control-variant': 'Events with the control variant received',
+            'no-test-variant': 'Events with at least one test variant received',
+        }
+
+        return (
+            <div className="flex items-center space-x-1">
+                {checked ? (
+                    <IconCheckbox className={`w-6 ${checked ? 'text-success' : ''}`} />
+                ) : (
+                    <IconSquare color="var(--muted)" fontSize="24" />
+                )}
+                <span>{failureReasonToText[failureReason]}</span>
+            </div>
+        )
+    }
 
     if (experimentResultsLoading) {
         return <></>
     }
 
+    if (experimentInsightType === InsightType.FUNNELS && experimentResultCalculationError) {
+        const checklistItems = []
+        for (const [failureReason, value] of Object.entries(JSON.parse(experimentResultCalculationError))) {
+            checklistItems.push(<ChecklistItem key={failureReason} failureReason={failureReason} checked={!value} />)
+        }
+
+        return (
+            <div>
+                <h2 className="font-semibold text-lg">Results</h2>
+                <div className="border rounded bg-bg-light pt-6 pb-8">
+                    <div className="flex space-x-2">
+                        <div className="w-1/2 py-4 px-6 space-y-4">
+                            <div className="font-semibold leading-tight text-lg text-current">
+                                Results not yet available
+                            </div>
+                            <div className="text-muted">
+                                Results will be calculated once we've received the necessary minimum events. The
+                                checklist on the right shows what's still needed.
+                            </div>
+                        </div>
+                        <LemonDivider vertical />
+                        <div className="w-1/2 flex py-4 px-6 items-center">
+                            <div className="space-y-2">{checklistItems}</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        )
+    }
+
     return (
         <div>
             <h2 className="font-semibold text-lg">Results</h2>
-            <div className="border rounded bg-bg-light pt-6 pb-8 text-muted">
-                <div className="flex flex-col items-center mx-auto">
+            <div className="border rounded bg-bg-light pt-6 pb-8">
+                <div className="flex flex-col items-center mx-auto text-muted">
```

Comment:
Another follow up: Sometimes there are 500 errors, like failure to calculate for whatever reason. Would be nice to make sure we show a reasonable error message then too.